code/src/components/Section.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Question } from './Question'
+import { Answer } from './Answer'
+import OkButton from './Button'
+import Progressbar from './Progressbar'
+
+const questions = [
+  'What is your name?', 'How do you feel about working from home?', 'What are you finding most difficult?', 'Do you have the following equipment at home?', 'Are you getting the support you need from your manager?']
+
```

Comment:
It might be a bit tricky to follow with the naming of components such as answers.js result.js and section.js? Might I suggest names that are a bit more specific in description?